esmvaltool/recipes/recipe_arctic_ocean.yml

```
@@ -0,0 +1,178 @@
+    scripts:
+      arctic_ocean_main:
+        script: arctic_ocean/arctic_ocean.py
+        # Define if we calculate data for Hovmoeller diagrams.
+        # This data are also needed for mean vertical profiles.
+        hofm_data: True
+        # Define regions, as a list.
+        # 'EB' - Eurasian Basin of the Arctic Ocean
+        # 'AB' - Amerasian Basin of the Arctic Ocean
+        # 'Barents_sea' - Barrents Sea
+        # 'North_sea'   - North Sea
+        hofm_regions: ["AB" ,  'EB']
+        # Define variables to use, should also be in "variables" entry of your diagnostic
+        hofm_vars: ['thetao', 'so']
+        # Maximum depth of Hovmoeller and vertical profiles
+        hofm_depth: 1500
+        # Define if Hovmoeller diagrams will be ploted.
+        hofm_plot: True
+        # Define colormap (as a list, same size as list with variables)
+        # Only cmaps from matplotlib and cmocean are supported.
+        # Additional cmap - 'custom_salinity1'.
+        hofm_cmap: ['Spectral_r', 'custom_salinity1']
+        # Data limits for plots,
+        # List of the same size as the list of the variables
+        # each entry is [vmin, vmax, number of levels, rounding limit]
+        hofm_limits: [[-2, 2.3, 41, 1], [30.5, 35.1, 47, 2]]
+        # Number of columns in the plot
+        hofm_ncol: 3
+        # Calculate the timemmean for every model
+        mean: True
+        # Plot vertical profiles
+        profiles: True
+        # Plot spatial distribution maps
+        plot2d: True
+        # Depths for spatial distribution maps
+        plot2d_depths: [10, 100]
+        # Variables to plot spatial distribution maps
+        plot2d_vars: ['thetao', 'so']
+        # Define colormap (as a list, same size as list with variables)
+        # Only cmaps from matplotlib and cmocean are supported.
+        # Additional cmap - 'custom_salinity1'.
+        plot2d_cmap: ['Spectral_r', 'custom_salinity1']
+        # Data limits for plots,
+        # List of the same size as the list of the variables
+        # each entry is [vmin, vmax, number of levels, rounding limit]
+        plot2d_limits: [[-2, 4, 20, 1], [30.5, 35.1, 47, 2]]
+        # number of columns for plots
+        plot2d_ncol: 4
+        # Plot spatial distribution of the bias plots
+        plot2d_bias: True
+        # Depths for spatial distribution of the bias
+        plot2d_bias_depths: [10, 100]
+        # Variables to plot spatial distribution of the bias for
+        plot2d_bias_vars: ['thetao', 'so']
+        # Color map bames for every variable
+        plot2d_bias_cmap: ['balance', 'balance']
+        # Data limits for plots,
+        # List of the same size as the list of the variables
+        # each entry is [vmin, vmax, number of levels, rounding limit]
+        plot2d_bias_limits: [[-3, 3, 20, 1], [-2, 2, 47, 2]]
+        # number of columns in the bias plots
+        plot2d_bias_ncol: 4
+        # Plot transects
+        transects: True
```

```suggestion
        transects: true
```
as suggested by `yamllint`